pkg/serviceaccounts/oauthclient/oauthclientregistry_test.go

```
@@ -1172,8 +1210,9 @@ func TestRedirectURIsFromRoutes(t *testing.T) {
 		},
 	} {
 		a := buildRouteClient(test.routes)
-		if !reflect.DeepEqual(test.expected, a.redirectURIsFromRoutes(test.namespace, test.names)) {
-			t.Errorf("%s: expected %#v, got %#v", test.name, test.expected, a.redirectURIsFromRoutes(test.namespace, test.names))
+		uris, _ := a.redirectURIsFromRoutes(test.namespace, test.names)
```

Don't ignore the errors.